v2/common/src/test/java/com/google/cloud/teleport/v2/utils/BeamSchemaUtilsTest.java

```diff
@@ -399,4 +399,17 @@ public void givenBeamSchema_whenNullableTrue_thenCorrectJsonString()
 
     Assert.assertEquals(mapper.readTree(expectedJson), mapper.readTree(beamSchemaJson));
   }
+
+  @Test
+  public void testNullableField() throws IOException, SchemaParseException {
+    String jsonString = "[{\"name\": \"nullableField\", \"type\": \"STRING\", \"nullable\": true},{\"name\": \"nonNullableField\", \"type\": \"STRING\", \"nullable\": false},{\"name\": \"defaultNullableValueField\", \"type\": \"STRING\"}]";
+    Schema schema = BeamSchemaUtils.fromJson(jsonString);
+    assertEquals(3, schema.getFieldCount());
+    assertEquals("nullableField", schema.getField(0).getName());
+    assertEquals(FieldType.STRING.withNullable(true), schema.getField(0).getType());
+    assertEquals("nonNullableField", schema.getField(1).getName());
+    assertEquals(FieldType.STRING, schema.getField(1).getType());
+    assertEquals("defaultNullableValueField", schema.getField(2).getName());
+    assertEquals(FieldType.STRING, schema.getField(2).getType());
+  }
 }
```

Comment:
Could you please also add an empty string at the end of this file? It is a convention 🙂